src/main/java/org/opentripplanner/routing/algorithm/transferoptimization/model/MinSafeTransferTimeCalculator.java

```diff
@@ -92,6 +99,17 @@ int durationIncludingSlack(TransitPathLeg<T> leg)  {
     if(list.isEmpty()) { return MIN_SAFE_TRANSFER_TIME_LIMIT_UPPER_BOUND; }
     int minTransitTime = list.stream().mapToInt(transitTimeSeconds).min().getAsInt();
     int minSafeTransitTime = (int) Math.round(minTransitTime * P / 100.0);
-    return Math.min(MIN_SAFE_TRANSFER_TIME_LIMIT_UPPER_BOUND, minSafeTransitTime);
+    return bound(
+            minSafeTransitTime,
+            MIN_SAFE_TRANSFER_TIME_LIMIT_LOWER_BOUND,
```

Comment:
This probably needs to be changed once support for defining the GTFS transfers.txt min_transfer_time is added?

Reply:
I am not sure if it has anything to do with each other. The `minSafeTransitTime` is used to add a cost for very short transfers _across all possible transfers for a set of trips_, while the GTFS `min_transfer_time` is a restriction for a given _pair of stops(& trips)_. The OptimizeTransferService and the Raptor search must both support the GTFS `min_transfer_time` - preventing transfers when appropriate.